seclang/directives.go

```diff
@@ -470,6 +471,28 @@ func directiveSecIgnoreRuleCompilationErrors(options *DirectiveOptions) error {
 	return nil
 }
 
+func directiveSecDataset(options *DirectiveOptions) error {
+	spl := strings.SplitN(options.Opts, " ", 2)
+	if len(spl) != 2 {
+		return errors.New("syntax error: SecDataset name `\n...\n`")
+	}
+	name := spl[0]
+	if _, ok := options.Datasets[name]; ok {
+		options.Waf.Logger.Warn("Dataset %s already exists, overwriting", name)
```

Comment:
I'd rather use `%q`, it is handy when debugging and an empty space is the cause of the problem.

Reply:
Sure, we should normalize using %q for external inputs. We should document it as technical debt